mlprimitives/jsons/sklearn.linear_model.Ridge.json

```diff
@@ -0,0 +1,106 @@
+            },
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "produce": {
+        "method": "predict",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "fixed": {
+            "copy_X": {
+                "type": "bool",
+                "description": "If True, X will be copied; else, it may be overwritten.",
+                "default": true
+            },
+            "tol": {
+                "type": "float",
+                "description": "Precision of the solution",
+                "default": 0.001
+            },
+            "max_iter": {
+                "type": "int",
+                "description": "Maximum number of iterations for conjugate gradient solver. For ‘sparse_cg’ and ‘lsqr’, the default is determined by scipy.sparse.linalg. For ‘sag’, the default value is 1000.",
+                "default": null
+            },
+            "random_state": {
+                "type": "multitype",
+                "description": "The seed of the pseudo random number generator to use when shuffling the data. Used when solver == 'sag'",
+                "types": {
+                    "int":{
+                        "description": "random_state is the seed used by the random number generator"
+                    },
+                    "RandomState":{
+                        "description": "If an array is passed, penalties are assumed to be specific to the targets. Hence they must correspond in number."
+                    },
+                    "NoneType": {
+                        "description": "If none, the random number generator is the RandomState instance used by np.random"
+                    }
+                },
+                "default": null
+            }
+        },
+        "tunable": {
+            "normalize": {
+                "type": "bool",
+                "default": false,
+                "description": "This parameter is ignored when fit_intercept is set to False. If True, the regressors X will be normalized before regression by subtracting the mean and dividing by the l2-norm."
+            },
+            "fit_intercept": {
+                "type": "bool",
+                "default": true,
+                "description" : "Whether to calculate the intercept for this model. If set to False, no intercept will be used in calculations (e.g. data is expected to be already centered)."
+            },
+            "solver": {
+                "type": "str",
+                "description": "Solver to use in the computational routines.",
+                "values": ["auto", "svd", "cholesky", "lsqr", "sparse_cg", "sag", "saga"],
+                "default": "auto"
+            },
+            "alpha": {
```

Comment:
Just like above, this should be set as a `float` hyperparameter and be given a reasonable range.